src/network/networkd-manager-varlink.c

```diff
@@ -81,6 +84,7 @@ static int dispatch_interface(sd_varlink *vlink, sd_json_variant *parameters, Ma
         static const sd_json_dispatch_field dispatch_table[] = {
                 { "InterfaceIndex", _SD_JSON_VARIANT_TYPE_INVALID, json_dispatch_ifindex,         offsetof(InterfaceInfo, ifindex), SD_JSON_RELAX },
                 { "InterfaceName",  SD_JSON_VARIANT_STRING,        sd_json_dispatch_const_string, offsetof(InterfaceInfo, ifname),  0             },
+                VARLINK_DISPATCH_POLKIT_FIELD,
```

Comment:
Some methods do not use polkit, e.g. `vl_method_get_lldp_neighbors()`.
Please make `dispatch_interface()` take a boolean argument, say `bool use_polkit`, then introduce two tables and conditionalize table to be used.